lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -84,6 +89,20 @@ export abstract class ClientApplication {
         return authorizationCodeClient.acquireToken(request);
     }
 
+    /**
+     * API to retrieve application cache
+     */
+    public getCache(): string {
+        return this.nodeStorage.getSerializedCache();
+    }
+
+    /**
+     * API to write cache to a file
+     */
+    public setCache(): void {
```

Comment:
Is this temporary? I think we shouldn't be providing an implementation of setCache(). We should provide an interface and the developer can provide the implementation.

Reply:
Well yes, until we finalize our extension library interface.